### ISSUE TITLE
Epic: JetBrains dedicated ports window

### ISSUE THREAD

**Issue by DRSchlaubi** [gitpod-io/gitpod#9700]
A dedicated ports view in JetBrains

Related issues: 
- https://github.com/gitpod-io/website/issues/2958

<details>
  <summary>Original issue</summary>

## Describe the behaviour you'd like
In VS Code their is a Gitpod tool window for ports, IntelliJ has UI for ports, however that only works on the local machine the client is installed on, if you want to use the Gitpod port forwarding, you need to use the CLI, which cannot forward public ports (afaik)

</details>

**Comment by loujaybee** [comment]
Thanks for raising @DRSchlaubi, we are currently discussing with JetBrains about this. We are expecting some updates to JetBrains Gateway etc that hopefully will make this possible in the next version 🙏

**Comment by akosyakov** [comment]
@ENate Could you file another issue? An error sounds like this workspace does not belong to you. Live workspace sharing is not supported for JB.

**Comment by ENate** [comment]
Thanks @akosyakov . You are right in saying that the workspace does not belong to me. But I am a collaborator to the Git repo of the project and will like to open the project on my Vscode desktop. It seems it is not possible to open the repo/workspace on my vscode desktop if I sign into github with another work account. Does this mean it is not possible to open this project (on my vscode desktop) using my work account (instead of a personal account) if I am added as a collaborator to the repo? Thank you.

**Comment by akosyakov** [comment]
@ENate Please move discussion to another issue. It is off-topic here. 🙏

**Comment by ENate** [comment]
Ok Thanks

**Comment by loujaybee** [comment]
A couple updates here, JetBrains have now worked on a dedicated ports view and we'll look at integrating this soon. 

Also, we've now got support for multiple terminals (showing for IDEs in latest currently:
- https://github.com/gitpod-io/gitpod/issues/10581

And we're still resolving some issues with showing the ports view on those terminals:
- https://github.com/gitpod-io/gitpod/issues/10579

In addition to the dedicated view, these should both help with generally exposing port visibility in the IDE.

**Comment by loujaybee** [comment]
**Currently blocked / waiting on JetBrains:** The ports view is not fully complete. We're waiting on API's for sorting ports, and also we would like to grouping and organise forwarded and exposed ports differently. However, we should consider breaking these out into separate bugs and issues, and consider this epic complete.

**Comment by loujaybee** [comment]
Still waiting to compile links for JetBrains on YouTrack.

**Comment by loujaybee** [comment]
Issues have been progressed as far as we can for now. We're waiting on some further updates from Jetbrains API's, however we'll open smaller issues to address. More information on the change in the changelog: 

https://www.gitpod.io/changelog#manage-workspace-ports-without-needing-to-leave-your-jetbrains-ui
